frontend/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import NavBar from "./Components/NavBar";
+import Home from "./Components/Home";
+import Brands from "./Components/Brands";
+import About from "./Components/About";
+import SearchResult from "./Components/SearchResult";
+import Details from "./Components/Details";
+import BrandListing from "./Components/BrandListing";
+
+
+function App() {
+  return (
+    <>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Home></Home>} />
```

Comment:
No need to write `<Home></Home>` you should write `<Home/>` for all below components, also add a `path="*"` for showing `404` component.